Sources/ServiceContextModule/ServiceContext.swift

```diff
@@ -233,8 +233,8 @@ extension ServiceContext {
 #if swift(>=5.5) && canImport(_Concurrency)
 @available(macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0, *)
 extension ServiceContext {
-    /// A `ServiceContext` automatically propagated through task-local storage. This API enables binding a top-level `ServiceContext` and passing it
-    /// implicitly to any child tasks when using structured concurrency.
+    /// A `ServiceContext` is propagated through task-local storage. This API enables binding a top-level `ServiceContext` and
```

Comment:
```suggestion
    /// A `ServiceContext` is automatically propagated through task-local storage. This API enables binding a top-level `ServiceContext` and
```

I feel like the "automatically" adds to the explanation.